### PR TITLE
added MonkeyLeague (MBS) token

### PR DESCRIPTION
**Pull request by tompere**

### CODE REVIEW

**Comment by zainbacchus** [comment]
cc @roberto-bayardo for review

**Comment by roberto-bayardo** [comment]
Thanks for this submission. We'll get back to this after next week when the team is back in the office.

**Comment by roberto-bayardo** [comment]
We'll take a look in the coming week.

**Comment by roberto-bayardo** [comment]
Validation is failing with the following reasons. Can you look into these?

```
error: MBS on chain goerli token 0x4fBb4BF42eE32A0dDDe8c531c958086A92A6980B has incorrect name
error: MBS on chain base-goerli token 0x5eBee9770969e9b2EA527A79924f461B2d252f8c has incorrect name
warning: MBS on chain ethereum token 0x4Bbf46E7c31eDc1F33aF4d57F0C4a0f13398427c not found on CoinGecko token list
error Command failed with exit code 1.
```

**Comment by roberto-bayardo** [comment]
> it happens because the token name was changed between goerli (3 weeks ago) and today's deployment on Ethereum mainnet. symbol remained the same.
> 
> is that an issue?

not necessarily -- just that I need to route tokens that fail validation for additional manual review.

**Comment by monkeytomer** [comment]
> 

It's possible, however, we will be launching all related utility features only when the Base blockchain will be live. At the moment MBS is supported only on Solana. We'd prefer to wait with this unless it is required.

File: data/MBS/data.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "MBS",
```
Comment (roberto-bayardo) [inline data/MBS/data.json:2]: Should this be "MonkeyLeague" instead of the token symbol?

File: data/MBS/data.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "symbol": "MBS",
+  "decimals": 18,
+  "description": "MonkeyLeague is the next-gen casual esports game empowering players to create, play, compete, and earn.",
+  "website": "https://monkeyleague.io",
+  "twitter": "@TheMonkeyLeague",
+  "tokens": {
+    "ethereum": {
+      "address": "0x4Bbf46E7c31eDc1F33aF4d57F0C4a0f13398427c"
```
Comment (roberto-bayardo) [inline data/MBS/data.json:10]: If you change the top-level name to "MonkeyLeague" you can add the following here to eliminate the validation error around mismatching name:
```
override: {
  name: "MonkeyLeague"
}
```

File: data/MBS/data.json (hunk 1 above)
Comment (tremarkley) [reply data/MBS/data.json:2]: This is typically set to the name, i.e. MonkeyLeague

File: data/MBS/data.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    },
+    "goerli": {
+      "address": "0x4fBb4BF42eE32A0dDDe8c531c958086A92A6980B"
+    },
```
Comment (roberto-bayardo) [inline data/MBS/data.json:14]: Now that base mainnet is launched to developers feel free to deploy there & add a "base" section to this PR while we're getting the rest of the issues figured out.

File: data/MBS/data.json (hunk 1 above)
Comment (tompere) [reply data/MBS/data.json:2]: it's by design, we renamed it to `MBS`(from  `MonkeyLeague`)

File: data/MBS/data.json (hunk 3 above)
Comment (tompere) [reply data/MBS/data.json:14]: 👍 will be done so on Sunday

File: data/MBS/data.json (hunk 2 above)
Comment (tompere) [reply data/MBS/data.json:10]: I tried it the other way around with overriding the goerli tokens with the top-level `MBS` name. will that work?

Comment (roberto-bayardo) [reply data/MBS/data.json:10]: yeah I think so.

**Review by roberto-bayardo — APPROVED** [review]

**Comment by roberto-bayardo** [comment]
looks good from the Base side!

**Review by roberto-bayardo — APPROVED** [review]

**Review by roberto-bayardo — APPROVED** [review]

**Comment by roberto-bayardo** [comment]
@nickbalestra ptal?

**Review by nickbalestra — APPROVED** [review]

**Pull request merged**